lib/ts/recipe/session/sessionClass.ts

```diff
@@ -272,15 +274,25 @@ export default class Session implements SessionContainerInterface {
         if (this.accessTokenUpdated) {
             const { res, transferMethod } = info;
 
-            setAccessTokenInResponse(res, this.accessToken, this.frontToken, this.helpers.config, transferMethod);
+            setAccessTokenInResponse(
+                res,
+                this.accessToken,
+                this.frontToken,
+                this.helpers.config,
+                transferMethod,
+                info.req,
+                {}
```

Comment:
why passing in {}? Pass the userContext as an optional param into this func signature (so this is not a breaking change for the SDK then). And then in this function, if the userContext is undefined, create a default one like it's done in the APIs.

Also, during impl, first make it compulsory, so that the types complain in all places where this function is called, and then make it optional